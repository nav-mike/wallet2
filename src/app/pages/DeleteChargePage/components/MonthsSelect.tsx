import { Select } from 'app/components/Select';
import styled from 'styled-components/macro';

export const MonthsSelect = styled(Select)`
  flex: 2;
`;
