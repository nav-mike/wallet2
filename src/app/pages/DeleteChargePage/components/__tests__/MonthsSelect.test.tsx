import * as React from 'react';
import { render } from '@testing-library/react';
import { MonthsSelect } from '../MonthsSelect';

describe('<MonthsSelect />', () => {
  it('should render and match the snapshot', () => {
    const select = render(<MonthsSelect />);
    expect(select.container.firstChild).toMatchSnapshot();
  });
});
