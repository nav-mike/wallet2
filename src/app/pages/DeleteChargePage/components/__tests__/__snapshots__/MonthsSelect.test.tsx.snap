// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MonthsSelect /> should render and match the snapshot 1`] = `
.c0 {
  border: 1px solid rgba(220,120,95,1);
  border-radius: 0.3rem;
  padding: 0.2rem;
  -webkit-flex: 2;
  -ms-flex: 2;
  flex: 2;
}

.c0:hover {
  cursor: pointer;
}

<select
  class="Select-sc-5sdtip-0 c0"
/>
`;
